@xen-orchestra/lite/src/views/vm/system/VmVirtualizationAndBoot.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <UiCard class="vm-virtualization-and-boot">
+    <UiTitle>
+      {{ $t('virtualization-boot-settings') }}
+    </UiTitle>
+    <VtsQuickInfoRow :label="$t('virtualization-mode')" :value="virtualizationMode" />
+    <VtsQuickInfoRow :label="$t('secure-boot')">
+      <template #value>
+        <VtsEnabledState :enabled="vm.platform.secureBoot !== undefined" />
+      </template>
+    </VtsQuickInfoRow>
+    <VtsQuickInfoRow :label="$t('virtual-tpm')" :value="vm.VTPMs.length > 0 ? vm.VTPMs.join(', ') : $t('none')" />
+    <VtsQuickInfoRow :label="$t('viridian')">
+      <template #value>
+        <UiInfo :accent="vm.platform.viridian === 'true' ? 'success' : 'muted'">
+          {{ vm.platform.viridian === 'true' ? $t('enabled') : $t('disabled') }}
+        </UiInfo>
```

Comment:
```suggestion
        <VtsEnabledState :enabled="vm.platform.viridian === 'true'" />
```
Remember to delete the `UiInfo` component import